test/helpers/feature-emulation.helpers.js

```diff
@@ -1079,6 +1079,14 @@
                 registerDefaultToStringAdapter(this, location, '[object Location]');
             patchGlobalToString(this);
         },
+        MOZILLA:
+        function ()
+        {
+            if (!global.navigator)
+                override(this, 'navigator', { value: { } });
+            var Mozilla = createStaticSupplier('Mozilla/5.0');
+            override(this, 'navigator.userAgent', { value: Mozilla });
```

Comment:
It seems that this feature will only need emulation in Node.js where `navigator` is not defined.

```suggestion
            var navigator = { userAgent: 'Mozilla/5.0' };
            override(this, 'navigator', { value: navigator });
```